filter/fes-2.0.2/src/main/java/org/orbisgis/orbisdata/filter/fes_2_0_2/FesToSql.java

```diff
@@ -90,11 +95,12 @@ public static StringBuilder XmlToSql(Object objectFromFilterXml) {
                                     returnSQL.append(property.getValueReference());
                                 }
                             }
-                        }
+                        }else logger.warn("Warning : there aren't any property between the tag SortBy from the xml file ");
```

Comment:
I don't understand what do you mean by "format". Is there an example ? because i used the example from OrbisWps for the object logger and the documentation from [internet](https://www.google.fr/url?sa=t&rct=j&q=&esrc=s&source=web&cd=1&cad=rja&uact=8&ved=0ahUKEwjOs8jtqZXUAhULuhoKHYaXA6MQFggmMAA&url=https%3A%2F%2Fwww.slf4j.org%2Fapidocs%2Forg%2Fslf4j%2FLogger.html&usg=AFQjCNE2hRzL6c7nIULR8BGIxkrAOUVqzA).